### PR TITLE
ADEN-9458 Fix missing labrador for garfield

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

File: extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js [hunk 1]
```diff
@@ -147,6 +152,15 @@ class BillTheLizardWrapper {
         return `incontent_boxad_${counter}`;
     }
 
+    hasAvailableModels(btlConfig, projectName) {
+        const projects = btlConfig.projects || config.projects;
```
Comment (jacekbj) [inline extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js]: Variable `config` does not exist.

**Review by jacekbj — CHANGES_REQUESTED** [review]
Please remove the redundant variable.

File: extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js (hunk 1 above)
Comment (preludek) [reply extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js]: 👍

**Review by jacekbj — APPROVED** [review]
Checking if geo is right in order to track sth is very obscure yet effective.

**Pull request merged**